membership-attribute-service/app/models/Attributes.scala

```diff
@@ -121,6 +126,7 @@ object Attributes {
     (__ \ "userId").write[String] and
       (__ \ "tier").writeNullable[String] and
       (__ \ "recurringContributionPaymentPlan").writeNullable[String] and
+      (__ \ "oneOffContributionDate").writeNullable[LocalDate].contramap[Option[LocalDate]](_ => None) and  //omit
```

Comment:
Why do you need the contramap here?

Reply:
to exclude this field from the api response by setting it to `None`